lib/jsonapi-serializers/serializer.rb

```diff
@@ -316,8 +316,25 @@ def self.serialize(objects, options = {})
       result
     end
 
-    def self.serialize_errors(errors)
-      {'errors' => errors}
+    def self.serialize_errors(raw_errors)
+      {'errors' => errors(raw_errors)}
+    end
+
+    def self.errors(raw_errors)
+      if raw_errors.class == ActiveModel::Errors
```

Comment:
Down with the ducks, now that you point it out. Explicit type check doesn't seem to be your style reading through the code.